integration/test/chart/basic/basic_test.go

```diff
@@ -0,0 +1,145 @@
+// +build k8srequired
+
+package basic
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+
+	"github.com/giantswarm/chart-operator/integration/key"
+)
+
+// TestChartLifecycle tests a Helm release can be created, updated and deleted
+// uaing a chart CR processed by chart-operator.
+//
+// - Create chart CR using apiextensions-chart-e2e-chart.
+// - Ensure test app specfied in the chart CR is deployed.
+//
+// - Update chart CR using apiextensions-chart-e2e-chart.
+// - Ensure test app is redeployed using updated chart tarball.
+//
+// - Delete apiextensions-chart-e2e-chart.
+// - Ensure test app is deleted.
+//
+func TestChartLifecycle(t *testing.T) {
+	ctx := context.Background()
+
+	// Test creation.
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chart %#q", key.CustomResourceReleaseName()))
+
+		c := chartvalues.APIExtensionsChartE2EConfig{
+			Chart: chartvalues.APIExtensionsChartE2EConfigChart{
+				Name:       key.TestAppReleaseName(),
+				Namespace:  "giantswarm",
+				TarballURL: "https://giantswarm.github.com/sample-catalog/kubernetes-test-app-chart-0.5.3.tgz",
+			},
+			ChartOperator: chartvalues.APIExtensionsChartE2EConfigChartOperator{
+				Version: "1.0.0",
+			},
+			Namespace: "giantswarm",
+		}
```

Comment:
I don't know if it makes sense to use charts to install CRs in test. I'd just use g8sClient.

Reply:
Yes it is kind of overkill. But what is the purpose of the charts in apiextensions then?

Reply:
Test setup. I.e. installing vault, cert-operator, certs CRs, the tested operator itself, encryption secret etc.

Reply:
Ah in apiextensions you ask. Yeah I think it was because awsconfig is so complex.

I admit you are right here.

Reply:
Also initially creating `awsconfig` was a part of the setup. We need to rethink this stuff.

Reply:
Yes I'm open to simplifying. Maybe we use apiextensions just for large CRs? Using a chart to install a chart CR that installs a chart is pretty damn complicated!

Reply:
Uncertain. This is confusing. Let's first sort the setup in general.

Reply:
So I should move the CR creation to setup?

Reply:
No no, I was talking in general. I think this is good how it is for now.